lib/Checkout/Common/AbstractQueryFilter.php

```diff
@@ -17,9 +17,9 @@ public function getEncodedQueryParameters()
                 if (!empty($value)) {
                     $url .= $key . "=";
                     $url .= $value instanceof DateTime ? urlencode(CheckoutUtils::formatDate($value)) : $value;
-                }
-                if ($key != array_keys($vars)[count($vars) - 1]) {
-                    $url .= "&";
+                    if ($key != array_keys($vars)[count($vars) - 1]) {
```

Comment:
This is a bugfix or so, because right now if the property is empty the query is formed like `/&&from=something` which is incorrect, we should only append & if the property is not null